src/Simulation/Simulators/SparseSimulatorDeprecated/README.md

```diff
@@ -0,0 +1,143 @@
+# Introduction
```

Comment:
I believe the entire folder SparseSimulatorDeprecated should be deleted, and the content of this file should be moved to the new location. Check if the info in this file is still up to date.